vendor/bat-native-ledger/src/bat_helper.cc

```diff
@@ -760,6 +761,12 @@ static bool ignore_ = false;
       visits_ = d["visits"].GetUint();
       percent_ = d["percent"].GetUint();
       weight_ = d["pubs_load_timestamp"].GetDouble();
+
+      if (d.HasMember("verified") && d["verified"].IsBool()) {
+        verified_ = d["verified"].GetBool();
+      } else {
+        verified_ = false;
+      }
```

Comment:
Shouldn't we add verified_ into saveToJson in https://github.com/brave/brave-core/blob/master/vendor/bat-native-ledger/src/bat_helper.cc#L768 and we could check HasMember and its type as others at above and assign value here?

Reply:
never mind, saw the saveToJson below, isn't this member always existed?

Reply:
no it doesn't exist and that's why I need to be careful with check so that we have fallback for older versions

Reply:
got it, thanks for the explanation.